### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by lengushuru**
Hello 👋 .

In this PR the following changes were introduced as per [these ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md)specification

- Create a `app.rb` file that will serve as your console app entry-point.

- defined the entry point, this will be a method called main that is invoked at the end of  file.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+require './app'
+
+class Main
+  def initialize
+    @app = App.new
+  end
+
```
Comment (anagudelogu) [inline main.rb:7]: - [x] You did a great job working on this milestone! 💯 However, there is some debug content getting printed to the console when I run your app:

image

Please make sure that you remove old `puts` and `prints` that were intended for testing purposes! 🤓 🔧

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    print 'Age:'
+    age = gets.chomp.to_i
+
+    print 'Has Parent permission? [Y/N]:'
+    parent_permission = gets.chomp.downcase
+
+    student = Student.new(age, @classroom, name, parent_permission: parent_permission == 'y')
+    @people << student
+    puts 'Student created successfully'
+  end
```
Comment (anagudelogu) [inline app.rb]: - [ ] I'm getting an error trying to create a student:

image

Please make sure to fix it! 🤓 🔧 


💡 _Pay special attention to how your `Student` class expects the parameters to be passed and how you're initializing them. See [named parameters](https://thoughtbot.com/blog/ruby-2-keyword-arguments#required-keyword-arguments)_

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+    print 'Age:'
+    age = gets.chomp.to_i
+
+    print 'Specialization:'
+    specialization = gets.chomp
+
+    @people << Teacher.new(age, specialization, name)
+    puts 'Teacher created successfully'
+  end
```
Comment (anagudelogu) [inline app.rb]: - [ ] The same happens when I try to create a Teacher:

image

Please consider looking into it! 🤓 🔧 

💡 _Pay special attention to how your `Teacher` class expects the parameters to be passed and how you're initializing them. See [named parameters](https://thoughtbot.com/blog/ruby-2-keyword-arguments#required-keyword-arguments)_

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @lengushuru, 
I'm Andres.

While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (lengushuru) [reply main.rb:7]: Thanks for reminder I will work on them straight away 💯

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your app meets all the changes requested by the previous code reviewer ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**